### ISSUE TITLE
Make name of competition available without having to grab /competitions.json in api

### ISSUE THREAD

**Issue by coder13** [ciscou/cubecomps-mobile#4]

**Comment by ciscou** [comment]
hey @coder13 !

I know I'm 1 year and a half too late but better late than never!

Please, check these URLs I've just added

https://m.cubecomps.com/api/v1/competitions
https://m.cubecomps.com/api/v1/competitions/3545
https://m.cubecomps.com/api/v1/competitions/3545/competitors/16
https://m.cubecomps.com/api/v1/competitions/3545/events/1/rounds/1

Also, if you're doing anything with cubecomps' API, please let me know

cheers!

**Comment by coder13** [comment]
Hey thanks! I've completely forgotten what code to update thanks to this now but thanks for doing it anyways!

It's http://kinchcomps.surge.sh/
It calculates the kinch ranking for everyone at a competition (relative to the winner of each round) and shows the overall winner. We did/used to use this to hand out gift cards to the top 3 people who didn't podium but we still find it fun to look at who was the overall winner of the comp.

It's all reliant on calling your api instead of doing my own parsing of cubecomps.
I had to do an unnecessary call to /competitions.json to grab the name of the comp when you viewed a comp's page.
